indexer/ethereumetl/cli/stream.py

```diff
@@ -54,8 +55,9 @@
 @click.option('-w', '--max-workers', default=lambda: environ.get("MAX_WORKERS", 5), show_default=True, type=int, help='The number of workers')
 @click.option('--log-file', default=lambda: environ.get("LOG_FILE"), show_default=True, type=str, help='Log file')
 @click.option('--pid-file', default=lambda: environ.get("PID_FILE"), show_default=True, type=str, help='pid file')
+@click.option('--table-prefix', default=lambda: environ.get("TABLE_PREFIX"), show_default=True, type=str, help='Table prefix for Clickhouse')
```

Comment:
not recommend, please use url parameter like 

url?table_prefix
it won't break the compatible on all the functions